test/lsp-clangd-test.el

```diff
@@ -102,6 +102,7 @@ and starts lsp. After the test BODY runs - tidy up."
      (setq lsp-clients-clangd-args actual-clangd-args)))
 
 (ert-deftest lsp-clangd-initialised-workspace ()
+  :tags '(no-win)
```

Comment:
Can you not do that? It should work on all platforms, so we should check on windows, too. I can have a look at the CI logs.